### PR TITLE
Add a Python test file

### PR DESCRIPTION
**Pull request by GTZL1**

### CODE REVIEW

File: integration-tests/api.py [hunk 1]
```diff
@@ -14,6 +14,15 @@ def register(username, password):
...
         return { "error" : res.status_code }
 
+def login(username, password):
+    data = { "username" : username, "password" : password }
+    url = server + "/auth"
+    res = requests.post(url, json = data)
+    if (res.status_code == 200):
+        return res.json()
+    else:
+        return { "error" : res.status_code }
```
Comment (alexandrepiveteau) [inline integration-tests/api.py]: Isn't it exactly what the `auth` method does ?

**Review by alexandrepiveteau — CHANGES_REQUESTED** [review]

File: integration-tests/api.py (hunk 1 above)
Comment (GTZL1) [reply integration-tests/api.py]: No, that's a login, not a register. This way you don't have to delete the moderator each time you execute the script

Comment (alexandrepiveteau) [reply integration-tests/api.py]: https://github.com/heig-PRO-b04/java-backend/wiki/APIv1.Endpoints#post-auth

Comment (alexandrepiveteau) [reply integration-tests/api.py]: (it's exactly the same as the code defined on lines 26-33)

File: integration-tests/testModifyQuestionAndAnswers.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import api
 import sys
 
-reg = api.register("aloy", "chieftain")
+reg = api.auth("aloy", "chieftain")
```
Comment (alexandrepiveteau) [inline integration-tests/testModifyQuestionAndAnswers.py]: I believe the username does not exist when the integration tests are run. Using `api.register` should do it 🙂

**Review by alexandrepiveteau — CHANGES_REQUESTED** [review]

File: integration-tests/testModifyQuestionAndAnswers.py (hunk 2 above)
Comment (GTZL1) [reply integration-tests/testModifyQuestionAndAnswers.py]: it's fixed

**Review by alexandrepiveteau — APPROVED** [review]

**Pull request merged**